pkg/pillar/types/zedroutertypes.go

```diff
@@ -1940,6 +1941,12 @@ type NetworkInstanceInfo struct {
 	// to this bridge and their statistics.
 	// We add statistics from all vifs while reporting to cloud.
 	VifMetricMap map[string]NetworkMetric
+
+	// Maintain a map of all access vlan ids to their counts, used by apps
+	// connected to this network instance.
+	VlanMap map[uint32]uint32
+	// Counts the number of trunk ports attached to this network instance
+	NumTrunkPorts uint32
```

Comment:
Should we add this to the API for Network Instance metrics (or info?) so we can see the counters from the controller?

Reply:
Makes sense. Let me see if it can be added to object logging also.

Reply:
done

Reply:
Submitted the api/proto changes. Will generate compiled proto changs after the api/proto looks ok.

Reply:
done